_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -12,7 +12,6 @@ accessibility_requirements:
     inapplicable: further testing needed
   wcag20:2.1.3: # Keyboard (No Exceptions) (AAA)
     forConformance: true
-    secondary: true
```

Comment:
Also not sure why this shouldn't be a **more strict** secondary requirement instead.